src/playasophy/wonderdome/mode/strip_eq.clj

```diff
@@ -0,0 +1,75 @@
+(ns playasophy.wonderdome.mode.strip_eq
+  (:require
+    [playasophy.wonderdome.geometry.sphere :as sphere]
+    [playasophy.wonderdome.mode.core :as mode]
+    [playasophy.wonderdome.util.color :as color]
+  )
+)
+
+; StripEQ will process audio signals and turn each strip into an indicator for
+; the strength of a range of frequencies over time.
```

Comment:
Looks like general style for comment-only lines is the double semicolon:

;; should be for a general comment about a block of code, ; should be used to add a comment to the end of a line of code.

https://clojuredocs.org/examples-styleguide